projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics.module.ts

```diff
@@ -0,0 +1,22 @@
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { NgModule } from '@angular/core';
+import { CmsConfig, ConfigModule } from '@spartacus/core';
+import { QualtricsComponent } from './qualtrics.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    HttpClientModule,
+    ConfigModule.withConfig(<CmsConfig>{
```

Comment:
Add `ConfigModule.withConfig(defaultQualtricsConfig),` - doesn't matter that you already have `ConfigModule.withConfig`, should work.